src/validation/ValidationApi.js

```diff
@@ -0,0 +1,29 @@
+const request = require("request-promise");
+const { isEmpty } = require("lodash");
+
+class ValidationApi {
+  constructor(validationApiUrl, http = request) {
+    this.validationApiUrl = validationApiUrl;
+    this.http = http;
+  }
+
+  async validate(json) {
+    const result = {
+      valid: true
+    };
+
+    const res = await this.http.post(this.validationApiUrl, {
+      body: json,
+      json: true
+    });
```

Comment:
Is it worth raising an issue against the validator repo to do proper [content negotiation](https://en.wikipedia.org/wiki/Content_negotiation) based on http headers, instead of some custom format?

Reply:
Yeah, I think there's improvements that could be made. But it would require some input from survey runner team since they are [using the service in a slightly different way to validate live schemas](https://github.com/ONSdigital/eq-survey-runner/blob/master/scripts/test_schemas.sh#L4).

Reply:
Oh, hang on I misread the code - the object is part of the `request` lib, not anything to do with the validation API. So no need to worry about it!

Reply:
Oh yeah, sorry, that `json: true` arg is an option of the request lib which says that the result is expected to be `application/json` format.

Reply:
ignore everything i've said here then haha